packages/typespec-ts/src/modular/buildCodeModel.ts

```diff
@@ -841,6 +843,8 @@ function emitBasicOperation(
   } else {
     bodyParameter = emitBodyParameter(context, httpOperation);
     // Flatten the body parameter if it is an anonymous model
+    const sdkOperation = getSdkType(operation);
```

Comment:
Operation is unlikely to type. In SDKType we could split all into different types and adopt into TCGC by need. And operations could be categoried into normal, LRO, paging ect or their generation could be parameters or responses. 

So `no new debt` means if we need something from TCGC, we need to replace relevant parts.

Reply:
We need to find the sweetspot between meaningful refactoring but keeping the changes as scoped as possible